docs/components/DateInputView.jsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component} from "react";
+
+import Example from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import {DateInput} from "src";
+
+export default class DateInputView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {value: null};
+  }
+
+  render() {
+    return (
+      <View title="DateInput" sourcePath="src/DateInput/DateInput.jsx">
+        <p>
+          DateInput is an input that allows the user to select dates from a DatePicker.
+        </p>
+        <Example>
+          <DateInput
+            label="label"
```

Comment:
The label is not getting associated with the `for={name}` of the `input` field for some reason.

Might want to double check that it works correctly.